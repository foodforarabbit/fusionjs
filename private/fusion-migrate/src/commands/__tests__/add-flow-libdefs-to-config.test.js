const path = require('path');
const pluginTester = require('../../utils/flowconfig-test.js');

const fixtureDir = path.join(
  __dirname,
  '../../__fixtures__/flowconfig-step-fixture/'
);
const plugin = require('../add-flow-libdefs-to-config.js');

pluginTester(fixtureDir, plugin);
