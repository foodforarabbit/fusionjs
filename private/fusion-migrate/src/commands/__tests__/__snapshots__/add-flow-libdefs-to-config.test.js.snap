// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add-flow-libdefs-to-config config: config 1`] = `
"[ignore]
.*/node_modules/.*[^(package)]\\\\.json$

[include]
./src/

[libs]
./some-lib-filepath

[lints]

[options]

[strict]


      ↓ ↓ ↓ ↓ ↓ ↓

[ignore]
.*/node_modules/.*[^(package)]\\\\.json$

[include]
./src/

[libs]
./node_modules/fusion-core/flow-typed/tape-cup_v4.x.x.js
./node_modules/fusion-core/flow-typed/npm/koa_v2.x.x.js
./node_modules/fusion-plugin-redux-action-emitter-enhancer/flow-typed/npm/redux_v3.x.x.js
./node_modules/fusion-plugin-react-redux/flow-typed/npm/redux_v3.x.x.js
./node_modules/fusion-plugin-rpc-redux-react/flow-typed/redux-reactors_v1.x.x.js
./node_modules/fusion-plugin-rpc-redux-react/flow-typed/npm/redux_v4.x.x.js
./some-lib-filepath

[lints]

[options]

[strict]
"
`;
